website/server/models/user/schema.js

```diff
@@ -500,6 +500,7 @@ let schema = new Schema({
       questStarted: {$type: Boolean, default: true},
       invitedQuest: {$type: Boolean, default: true},
       majorUpdates: {$type: Boolean, default: true},
+      partyActivity: {$type: Boolean, default: true},
```

Comment:
Hmm the settings page on the website doesn't support push notifications settings that don't have an email setting so it'll need a small change

Reply:
Do we want/need the setting there? Without a mobile device, the setting is useless anyway. I don't think making the list of notifications longer on the website would help.

Reply:
right! If we don't care about having it on web (and indeed it doesn't really make sense) that's fine